doc/cephfs/quota.rst

```diff
@@ -74,6 +74,12 @@ To set a quota::
   setfattr -n ceph.quota.max_bytes -v 100000000 /some/dir     # 100 MB
   setfattr -n ceph.quota.max_files -v 10000 /some/dir         # 10,000 files
 
+``ceph.quota.max_bytes`` can also be set using human-friendly units
+(KiB, MiB, GiB, TiB)::
+
+  setfattr -n ceph.quota.max_bytes -v 100k /some/dir          # 100 MiB
```

Comment:
nit: the comment against `100k` is `100 MiB`
please make the correction in the comment or the actual example

Reply:
done